examples/minimal/packages/contracts/script/PostDeploy.s.sol

```diff
@@ -6,10 +6,11 @@ import { console } from "forge-std/console.sol";
 import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
 import { ResourceId, WorldResourceIdLib, WorldResourceIdInstance } from "@latticexyz/world/src/WorldResourceId.sol";
 import { RESOURCE_SYSTEM } from "@latticexyz/world/src/worldResourceTypes.sol";
+import { Systems } from "@latticexyz/world/src/codegen/tables/Systems.sol";
 
 import { MessageTable } from "../src/codegen/index.sol";
 import { IWorld } from "../src/codegen/world/IWorld.sol";
-import { ChatNamespacedSystem } from "../src/systems/ChatNamespacedSystem.sol";
+import { namespace__ChatNamespacedSystem } from "../src/systems/namespace__ChatNamespacedSystem.sol";
```

Comment:
we could probably just name this `namespace__ChatSystem` since it's clear from the name that it is namespaced

Reply:
True, that's better now